schema/codelists/tenderStatus.csv

```diff
@@ -1,8 +1,8 @@
-Code,Title,Description
-planning,Planning,"A future contracting process is being considered. Early information about the process can be provided in the tender section. A process with this status might provide information on early engagement or consultation opportunities, during which the details of a subsequent tender can be shaped."
-planned,Planned,"A contracting process is scheduled, but is not yet taking place. Details of the anticipated dates can be provided in the tender block."
-active,Active,A tender process is currently taking place.
-cancelled,Cancelled,The tender process has been cancelled.
-unsuccessful,Unsuccessful,The tender process was unsuccessful.
-complete,Complete,The tender process is complete.
-withdrawn,Withdrawn,No further information on this process is available under this ocid.
+Code,Title,Description,Deprecated,Deprecation note
+planning,Planning,"A future contracting process is being considered. Early information about the process can be provided in the tender section. A process with this status might provide information on early engagement or consultation opportunities, during which the details of a subsequent tender can be shaped.",1.2,"This code has been deprecated, because it is unnecessary since contracting processes are split from planning processes (which do not have any states themselves)."
+planned,Planned,"The contracting process is scheduled, but contracting documents (for example, planning documents) are not yet available to potential suppliers.",1.2,Deprecated because the same information can be determined by comparing `tender/datePublished` (new in 1.2) to `date` (or its absence)
+active,Active,"The contracting documents (for example, procurement documents) are available to potential suppliers, and the bid submission deadline has not yet passed.",1.2,Deprecated because the same information can be determined by comparing `tender/datePublished` (new in 1.2) and `tender/tenderPeriod/endDate` to `date`.
+complete,Complete,The bid submission deadline has passed.,1.2,Deprecated because the same information can be determined by comparing `tender/tenderPeriod/endDate` to `date`
+cancelled,Cancelled,"The contracting process was cancelled by the buyer or the procuring entity (for example, because of a change in needs, insufficient funds, or technical or procedural errors) after the contracting documents (for example, procurement documents) were available to potential supppliers, but not later than the bid submission deadline.",1.2,OCDS 1.2 replaces this code with a new field for final states.
```

Comment:
```suggestion
cancelled,Cancelled,"The contracting process was cancelled by the buyer or the procuring entity (for example, because of a change in needs, insufficient funds, or technical or procedural errors) after the contracting documents (for example, procurement documents) were available to potential suppliers, but not later than the bid submission deadline.",1.2,OCDS 1.2 replaces this code with a new field for final states.
```